analyseCI.sh

```diff
@@ -4,7 +4,7 @@ if [ "${0:0:1}" == "/" ]; then script_dir="$(dirname "$0")"; else script_dir="$(
 
 cd $script_dir
 
-if [ $# -ne 2 ]
```

Comment:
Why changing to 3 ? what is the third argument ? In the [readme ](https://github.com/YunoHost/CI_package_check/blob/master/README.md) there is still 2 arguments to `/PATH/analyseCI.sh`.

When I launch something like:
`/home/CI_package_check_buster/analyseCI.sh "/home/package_check/peertube_ynh" "peertube_yalh76_buster_testing"` I have as result : `This script need to take in argument the package which be tested and the name of the test.`

Reply:
Erh yeah indeed .. the third arg is supposed to be the test id provided by yunorunner .. it's in fact only used to later create a nice link in the XMPP notification (e.g. ci-apps-foobar.yunohost.org/job/$job_id). As a temporary workaround, you can feed it empty string and that should be okay